crates/autopilot/src/auction_transaction.rs

```diff
@@ -1,171 +0,0 @@
-//! This module is responsible for associating auction ids with transaction
```

Comment:
Renamed to `on_settlement_event_updater.rs` and extended.

Reply:
Maybe, in order to see the diff more easily, we could skip the file rename here and just create an issue for later to rename the file. It would make things much simpler to review (given the already very large set of changes).